kpops/components/streams_bootstrap/producer/producer_app.py

```diff
@@ -93,6 +93,15 @@ def add_extra_output_topic(self, topic: KafkaTopic, role: str) -> None:
     def helm_chart(self) -> str:
         return f"{self.repo_config.repository_name}/{AppType.PRODUCER_APP.value}"
 
+    async def reset(self, dry_run: bool) -> None:
+        """Not supported. Producer App cannot be reset.
```

Comment:
I think we need to reword this a little, e.g. producer app will only be destroyed. Reset is not necessary since it doesn't have consumer group offsets.